scipy/stats/_qmc.py

```diff
@@ -914,13 +910,38 @@ class LatinHypercube(QMCEngine):
 
         .. versionadded:: 1.8.0
 
+    orthogonal : bool, optional
+        Orthogonal array based LHS of strength 2 [7]_, [8]_.
+        When used, only ``n=p**2`` points can be sampled, with ``p`` a prime
+        number. It also constrains ``d <= p + 1``.
+        Default is False.
+
+        .. versionadded:: 1.8.0
+
     seed : {None, int, `numpy.random.Generator`}, optional
         If `seed` is None the `numpy.random.Generator` singleton is used.
         If `seed` is an int, a new ``Generator`` instance is used,
         seeded with `seed`.
         If `seed` is already a ``Generator`` instance then that instance is
         used.
 
+    Notes
+    -----
+
+    LHS is extremely effective on integrands that are nearly additive [2]_.
+    LHS on :math:`n` points never has more variance than plain MC on
+    :math:`n-1` points [3]_. There is a central limit theorem for LHS [4]_,
+    but not necessarily for optimized LHS.
```

Comment:
I'm having trouble understanding these three facts, what the connection is between them, and how they help me as a user of this function.

For instance, 
> LHS is extremely effective on integrands that are nearly additive

This first statement seems to be thinking in terms of using the LHS points as quadrature points. The connection between using LHS for, say, 1) design of a physical biological experiment and 2) numerically integrating a function may not be immediately obvious to users. Can the connection be more explicit?

> LHS on :math:`n` points never has more variance than plain MC

The variance of what quantity? I'm going to guess this actually means the variance of the value of the integral computed using Latin hypercube sampled points, but this sentence could potentially be interpreted to be about the variance of the coordinates of the LHS points themselves because it says "LHS... never has more variance". I would think that "Latin hypercube sampling" or "Latin hypercube sample" (whatever "LHS" stands for here) usually refers to the process of selecting the points, but here it seems to refer to computing something else with those points.

> There is a central limit theorem for LHS [4]_, but not necessarily for optimized LHS.

Again, I think this means that if an integral is estimated using LHS quadrature points, the values (on repeated calculation using fresh Latin hypercube samples) will be normally distributed. (And the previous fact says something about the variance of that distribution, right?) However, if we optimize the LHS, then this property may be lost, presumably because the samples are not so random anymore?

Reply:
I will revisit this part of the doc. It is only targeted as function integration as you guessed right and I agree that it can be confusing. (Side note, this is already in the doc, I just moved it down.)

Reply:
Let me know what you think of my edits.